src/routes/chains/entities/backbone.entity.ts

```diff
@@ -13,7 +13,7 @@ export class Backbone implements DomainBackbone {
   @ApiProperty()
   secure!: boolean;
   @ApiProperty()
-  settings!: Record<string, string>;
+  settings!: Record<string, unknown>;
```

Comment:
The `settings` are [not only `string`s](https://safe-client.safe.global/v1/chains/1/about/backbone).